extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1547,6 +1559,26 @@ export class AmpA4A extends AMP.BaseElement {
    */
   emitLifecycleEvent(unusedEventName, opt_extraVariables) {}
 
+  tryExecuteRealTimeConfig_() {
+    if (!!AMP.maybeExecuteRealTimeConfig) {
+      try {
+        return AMP.maybeExecuteRealTimeConfig(
+            this, this.getCustomRealTimeConfigMacros_());
+      } catch (err) {
+        dev().error(TAG, `Could not perform Real Time Config. Error: ${err}`);
```

Comment:
just pass err as the 3rd argument, logging has special code to handle it.  This should be user level to ensure it appears in the console